### PR TITLE
[FIX][account_user_audit]Fixed repaired problem, TASK:3157, ISSUE:558

### PR DESCRIPTION
**Pull request by hugho-ad**
about hidden menus when installed this module

### CODE REVIEW

**Comment by JulioSerna** [comment]
:+1:

**Comment by hugho-ad** [comment]
@JulioSerna ,
 please help me giving me a "+ 1", I need merge this pr because travis in the next pr , doesn't know the changes
Module in OML https://github.com/Vauxoo/odoo-mexico-v2/pull/160
Module in odoo-ifrs https://github.com/Vauxoo/odoo-ifrs/pull/31
Module in odoo-afr https://github.com/Vauxoo/odoo-afr/pull/19
new dependencies en lodi pr https://github.com/Vauxoo/lodigroup/pull/160

**Comment by JulioSerna** [comment]
:+1:

**Comment by hugho-ad** [comment]
@dsabrinarg , me apoyas con el merge

File: account_group_auditory/__openerp__.py [hunk 1]
```diff
@@ -45,7 +45,7 @@
     "license": "",
     "depends": [
         "account_asset",
-        "ifrs_report",
+        "account_voucher",
     ],
```
Comment (hbto) [inline account_group_auditory/__openerp__.py:49]: this module shall only depend on account, account_accountant

if there are needs to provide another modules then new bridge module shall be develop
that must depent on account_group_auditory

**Comment by hugho-ad** [comment]
@nhomar , @hbto , @JulioSerna , all this is ready, I got a dev instance
and the runbot green

http://dev.lodigroup.com:34069/

http://runbot.vauxoo.com/runbot/team/lodi-7#8_0-lodigroup-account-hugho-mod

**Comment by nhomar** [comment]
@hugho-ad merged.

**Pull request merged**